content/docs/releases/README.md

```diff
@@ -25,9 +25,9 @@ should be stable enough to run.
 
 | Release      | Release Date | End of Life            | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
 |--------------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
-| [1.14][]     | Feb 3, 2024  | Release of 1.16        | 1.24 → 1.29                        | 4.11 → 4.15                       |
-| [1.13][]     | Sep 12, 2023 | Release of 1.15        | 1.23 → 1.29                        | 4.10 → 4.15                       |
-| [1.12 LTS][] | May 19, 2023 | May 19, 2025           | 1.22 → 1.29                        | 4.9 → 4.15                        |
+| [1.15][]     | Feb 3, 2024  | Release of 1.17        | 1.24 → 1.30                        | 4.11 → 4.15                       |
```

Comment:
@ThatsMrTalbot the release data should be updated too.

Reply:
Also, some other items on this page need changing (eg. upcoming releases and old releases)